### ISSUE TITLE
eslint extension doesn't work

### ISSUE THREAD

**Issue by dennisjac** [Microsoft/vscode-eslint#414]
I'm running VSCode 1.20.1 and vscode-eslint 1.4.6 and while the plugin is installed and enabled when I open a directory with a valid .eslintrc the javascript code doesn't seem to get linted at all. The problems pane always shows no errors and the output and debug console panes stay empty.
Running eslint (globally installed) on the command line shows the problems correctly as expected.

**Comment by dennisjac** [comment]
I was able to "fix" this problem by creating a backup of the  `.config/Code` folder and then deleting it. Now the plugin seems to work but doesn't find the globally installed (`npm install -g`) eslint. In the terminal I can start eslint fine:
```
[dennis@nexusm master]$ eslint -v
v4.18.1
```

**Comment by dbaeumer** [comment]
@dennisjac this works for me. When you open the workspace can you switch to the ESLint Output channel and see if there is any error reported there?

**Comment by dennisjac** [comment]
This is what I see in the output pane:
```
[Info  - 12:17:48 AM] ESLint server is running.
[Info  - 12:17:49 AM] 
Failed to load the ESLint library for the document /var/www/html/master/client/components/directive/assets-table/assets-table.directive.js

To use ESLint please install eslint by running 'npm install eslint' in the workspace folder client
or globally using 'npm install -g eslint'. You need to reopen the workspace after installing eslint.

If you are using yarn instead of npm set the setting `"eslint.packageManager": "yarn"`

Alternatively you can disable ESLint for the workspace folder client by executing the 'Disable ESLint' command.
```
While `eslint` is globally installed and working fine when invoked manually the extension simply doesn't seem to recognize it.

**Comment by dbaeumer** [comment]
@dennisjac which package manager are you using? npm or yarn. For yarn you need to let eslint know about is using "eslint.packageManager": "yarn"`

**Comment by dennisjac** [comment]
I'm using npm and I've installed eslint with`sudo npm install -g eslint`. eslint is present in the system path and can be run normally:
```
[dennis@nexusm master]$ which eslint
/usr/bin/eslint
[dennis@nexusm master]$ eslint -v
v4.18.1
```

**Comment by dbaeumer** [comment]
What does `npm config get prefix` evalute to (execute in a terminal). This is where ESLint expects the node module to be installed. Is this true on our system ?

**Comment by dennisjac** [comment]
The problem was me apparently. In the past I installed nvm but forgot about it so I was working by default in an nvm environment but didn't realize it. So when I did `sudo npm install -g eslint` the eslint package ended up in the gobal system path which is not identical with the nvm environments.
`npm install -g eslint` (without the sudo) fixes the issue and now VSCode and the extension see the eslint package. It was the `npm config get prefix` that alerted me to this.

Sorry for the trouble.
